Sources/SwiftyXPC/XPCFileDescriptor.swift

```diff
@@ -19,4 +26,12 @@ public final class XPCFileDescriptor: Codable {
     public init(fileDescriptor: Int32) {
         self.fileDescriptor = fileDescriptor
     }
+    
+    public func dup() -> Int32 {
+        Darwin.dup(fileDescriptor)
+    }
+    
+    deinit {
+        close(fileDescriptor)
```

Comment:
This is probably an eccentricity of mine, but I like to use explicit `self.` in my personal style, so that it's always obvious what is a method/property as opposed to a local variable.

```suggestion
        close(self.fileDescriptor)
```